plugins/com.google.cloud.tools.eclipse.appengine.validation/src/com/google/cloud/tools/eclipse/appengine/validation/AppEngineWebMarkerResolutionGenerator.java

```diff
@@ -47,6 +45,12 @@
           .equals(marker.getType())) {
         IMarkerResolution fix = new VersionQuickFix();
         markerResolutions[0] = fix;
+      } else if ("com.google.cloud.tools.eclipse.appengine.validation.runtimeMarker"
+          .equals(marker.getType())) {
+        IMarkerResolution fix = new UpgradeRuntimeQuickFix();
```

Comment:
I saw a test failure. Should this be `UpgradeRuntimeSourceQuickFix` (unsure)?

Reply:
fixed